HW7/hw_7_task_1.py

```diff
@@ -0,0 +1,76 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+#
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+phone_book = [{"name": "Stepan Bandera", "phone": "+380501112233"},
+              {"name": "Taras Schevchenko", "phone": "+380672223344"},
+              {"name": "Yevhen Konovalets", "phone": "+380663332255"},
+              {"name": "Lina Kostenko", "phone": "+380634441166"},
+              {"name": "Bohdan Khmelnytskii", "phone": "+380675556677"}]
+
+# function for calculation number of elements
+def com_stats(phone_book):
+    elements_number = len(phone_book)
+    """return count of phone numbers"""
+    return elements_number
+
+# function for adding elements
+def com_add(phone_book):
+    new_contact = input(f"Please, enter name, surname and phone number (use space between): ")
+    name, surname, phone = new_contact.split()
+    """add a new contact to the phone book"""
+    phone_book.append({"name": name + " " + surname, "phone": phone})
+    """check if there is the same contact"""
+    if phone_book.count({"name": name + " " + surname, "phone": phone}) != 1:
+        print(f"This contact already exists!")
+    else:
+        return phone_book
```

Comment:
nit: use comments about function using """ under the function definition.
in other places of code use # comments
```suggestion
    """add a new contact to the phone book"""
    new_contact = input(f"Please, enter name, surname and phone number (use space between): ")
    name, surname, phone = new_contact.split()
    phone_book.append({"name": name + " " + surname, "phone": phone})
    # check if there is the same contact
    if phone_book.count({"name": name + " " + surname, "phone": phone}) != 1:
        print(f"This contact already exists!")
    else:
        return phone_book
```